orchestra/bots/tests/test_staffbot.py

```diff
@@ -31,39 +31,3 @@ def test_commands(self):
         mock_slack_data['text'] = 'invalid command'
         response = bot.dispatch(mock_slack_data)
         self.assertTrue(bot.default_error_text in response.get('text', ''))
-
-    def test_staff_command(self):
```

Comment:
i prefer having staffing unit tests here and leaving the view tests as simple ones, the view code is simple so the tests should cover the basic cases of user input it can accept and the staffbot tests validate the more complex staffing logic